fabric/v4/model_cloud_router_action_type.go

```diff
@@ -3,18 +3,17 @@
  *
  * Equinix Fabric is an advanced software-defined interconnection solution that enables you to directly, securely and dynamically connect to distributed infrastructure and digital ecosystems on platform Equinix via a single port, Customers can use Fabric to connect to: </br> 1. Cloud Service Providers - Clouds, network and other service providers.  </br> 2. Enterprises - Other Equinix customers, vendors and partners.  </br> 3. Myself - Another customer instance deployed at Equinix. </br>
  *
- * API version: 4.9
+ * API version: 4.10
  * Contact: api-support@equinix.com
  * Generated by: Swagger Codegen (https://github.com/swagger-api/swagger-codegen.git)
  */
 package v4
 
-// GatewayActionState : Cloud Router action state
-type GatewayActionState string
+// CloudRouterActionType : Cloud Router action type
+type CloudRouterActionType string
 
-// List of GatewayActionState
+// List of CloudRouterActionType
 const (
-	DONE_GatewayActionState    GatewayActionState = "DONE"
-	FAILED_GatewayActionState  GatewayActionState = "FAILED"
-	PENDING_GatewayActionState GatewayActionState = "PENDING"
+	BGP_SESSION_STATUS_UPDATE_CloudRouterActionType CloudRouterActionType = "BGP_SESSION_STATUS_UPDATE"
+	ROUTE_TABLE_ENTRY_UPDATE_CloudRouterActionType  CloudRouterActionType = "ROUTE_TABLE_ENTRY_UPDATE"
```

Comment:
My understanding is that this is a new/beta feature and breaking changes are expected at any time, but in general, in case something like this change is planned for a production feature:  enum changes like this are breaking for existing generated clients.  A client that strictly follows the spec will throw errors when new values start coming back for enum fields, and any customer code based on existing releases of this SDK will be broken while they wait for us to publish a new spec and generate/release an SDK update.  If we want customers to depend on our SDKs and associated tools, we need to establish a timeline for changes like these, communicate that to customers, and ensure that we support old & new clients for some period of time.  (It's possible that's already happening for production features and only happening here because there are no customers depending on these endpoints yet; I'm just calling out that this sort of change can't be standard procedure.)